app/src/main/java/com/github/fribourgsdp/radio/MainActivity.kt

```diff
@@ -25,10 +22,10 @@ class MainActivity : AppCompatActivity() {
         val button : Button = findViewById(R.id.button)
         button.setOnClickListener {startActivity(Intent(this, DisplayLyricsActivity::class.java))}
         val profileButton: ImageButton = findViewById(R.id.profileButton)
-        profileButton.setOnClickListener {val intent : Intent = Intent(this, UserProfileActivity::class.java).apply {
-                putExtra(USERNAME, "Default")
-            }
-            startActivity(intent)
+        profileButton.setOnClickListener {
+            startActivity(Intent(this, UserProfileActivity::class.java))
         }
+
+        User("bla", User.generateColor()).save(this)
```

Comment:
Is this line still necessary ?

Reply:
this line is a way to demo that saving and loading works because the profile page now fetches from device storage. It can be replaced by anyone working on the demo as this PR is purely for adding backend features

Reply:
Ok great !